src/sentry/integrations/jira_server/integration.py

```diff
@@ -494,7 +494,7 @@ def get_issue_url(self, key, **kwargs):
         return "{}/browse/{}".format(self.model.metadata["base_url"], key)
 
     def get_persisted_default_config_fields(self) -> Sequence[str]:
-        return ["project", "issuetype", "priority", "labels"]
+        return ["project", "issuetype", "labels"]
```

Comment:
This list specifies field names that should have their last used values persist when changing projects